tests/legacy/ApiTest.php

```diff
@@ -2308,12 +2309,13 @@ public function testApiConvertItemWithoutBody()
 			[
 				'network' => 'feed',
 				'title'   => 'item_title',
+				'uri-id'  => 1,
 				'body'    => '',
 				'plink'   => 'item_plink'
 			]
 		);
-		self::assertEquals('item_title', $result['text']);
-		self::assertEquals('<h4>item_title</h4><br>item_plink', $result['html']);
+		self::assertEquals("item_title\n\nParent status", $result['text']);
+		self::assertEquals('<h4>item_title</h4><br>Parent statusitem_plink', $result['html']);
```

Comment:
@annando - Are these changes really intended?

Reply:
If possible, we now store media as attached data. In the API this is then added to the body again. This is why you now have to provide an `uri-id`. And depending upon the value of the `uri-id` different attachments are added. If you don't want to have added something, you should use an `uri-id` that hasn't been used before.

Reply:
I see, thanks!